itest/rust/src/lib.rs

```diff
@@ -19,4 +19,9 @@ mod register_tests;
 // Entry point
 
 #[gdextension(entry_point=itest_init)]
-unsafe impl ExtensionLibrary for framework::IntegrationTests {}
+unsafe impl ExtensionLibrary for framework::IntegrationTests {
+    fn on_level_init(level: InitLevel) {
+        // Testing that we can use `SomeObject` during Servers init level. See `object_tests::init_level_test`.
```

Comment:
I wouldn't mention `SomeObject` here, that's an implementation detail and doesn't tell the reader what it is about. Instead, I would mention what is being tested.